tsconfig.json

```diff
@@ -1,77 +1,114 @@
-        "strictNullChecks": true /* Enable strict null checks. */,
-        "strictFunctionTypes": true /* Enable strict checking of function types. */,
-        "strictBindCallApply": true /* Enable strict 'bind', 'call', and 'apply' methods on functions. */,
-        "strictPropertyInitialization": true /* Enable strict checking of property initialization in classes. */,
-        "noImplicitThis": true /* Raise error on 'this' expressions with an implied 'any' type. */,
-        "alwaysStrict": true /* Parse in strict mode and emit "use strict" for each source file. */,
-        /* Additional Checks */
-        "noUnusedLocals": true /* Report errors on unused locals. */,
-        "noUnusedParameters": true /* Report errors on unused parameters. */,
-        "noImplicitReturns": true /* Report error when not all code paths in function return a value. */,
-        "noFallthroughCasesInSwitch": true /* Report errors for fallthrough cases in switch statement. */,
-        "noUncheckedIndexedAccess": false /* Include 'undefined' in index signature results */,
-        /* Module Resolution Options */
-        "moduleResolution": "node" /* Specify module resolution strategy: 'node' (Node.js) or 'classic' (TypeScript pre-1.6). */,
-        "esModuleInterop": true /* Enables emit interoperability between CommonJS and ES Modules via creation of namespace objects for all imports. Implies 'allowSyntheticDefaultImports'. */,
-        // "preserveSymlinks": true,              /* Do not resolve the real path of symlinks. */
-        "resolveJsonModule": true,
-        // "allowUmdGlobalAccess": true,          /* Allow accessing UMD globals from modules. */
-        /* Source Map Options */
-        // "sourceRoot": "",                      /* Specify the location where debugger should locate TypeScript files instead of source locations. */
-        // "mapRoot": "",                         /* Specify the location where debugger should locate map files instead of generated locations. */
-        // "inlineSourceMap": true,               /* Emit a single file with source maps instead of having a separate file. */
-        // "inlineSources": true,                 /* Emit the source alongside the sourcemaps within a single file; requires '--inlineSourceMap' or '--sourceMap' to be set. */
-        /* Experimental Options */
-        // "experimentalDecorators": true,        /* Enables experimental support for ES7 decorators. */
-        // "emitDecoratorMetadata": true,         /* Enables experimental support for emitting type metadata for decorators. */
-        /* Advanced Options */
-        "skipLibCheck": true /* Skip type checking of declaration files. */,
-        "forceConsistentCasingInFileNames": true /* Disallow inconsistently-cased references to the same file. */
-    }
+  "include": [
+    "./src/**/*",
+    "./*.tsx",
+    "./*.ts",
+    "./*.yml",
```

Comment:
Yaml files?

Reply:
Yup, else would just show file does not exist

Reply:
Checked, no need for it to go through ts / removed